packs/feats/combat-grab.json

```diff
@@ -24,7 +24,27 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "combat-grab",
+                "toggleable": true
+            },
+            {
+                "key": "Note",
+                "outcome": [
+                    "success",
+                    "criticalSuccess"
+                ],
+                "predicate": [
+                    "combat-grab"
+                ],
+                "selector": "strike-attack-roll",
+                "text": "PF2E.SpecificRule.Fighter.CombatGrab.Note",
+                "title": "PF2E.SpecificRule.Fighter.CombatGrab.Name"
```

Comment:
These should probably be `"title": "{item|name}"`.

Reply:
Also note that previously a similar PR was rejected:
https://github.com/foundryvtt/pf2e/pull/7386

Reply:
Good context
I added this because I have a player who has the Wrestler dedication who uses these two abilities and they NEVER post the abilities on chat.

Out of 5 players, I have 3 that never post their abilities to chat. The two that do are spellcasters...